install.sh

```diff
@@ -423,6 +423,10 @@ elif ! $packaging; then
     chown -R scylla:scylla $rdata
     chown -R scylla:scylla $rhkdata
 
+    for file in dist/common/sysctl.d/*.conf; do
+        bn=$(basename "$file")
+        sysctl -p "$rusr"/lib/sysctl.d/"$bn"
+    done
```

Comment:
Maybe it should be in post_install.sh? Or maybe we should merge post_install.sh here.